src/options.cpp

```diff
@@ -1556,7 +1563,7 @@ void options_manager::add_options_graphics()
 #   else
          cata_tiles::build_renderer_list(),
 #   endif
-    "software", COPT_CURSES_HIDE );
+    default_renderer, COPT_CURSES_HIDE );
```

Comment:
I suggest to move `default_renderer` logic to `cata_tiles::build_renderer_list`, rearranging this collection in such way that first value would be default for required system (e.g. `direct3d` for Windows tiles and `software` for other systems). Then we'll just take first value from this collection here.